.pre-commit-config.yaml

```diff
@@ -12,6 +12,11 @@ repos:
     hooks:
       - id: flake8
         language_version: python3
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.25.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py36-plus"]
```

Comment:
Should be `--py37-plus` (though, in practice, that doesn't seem to change anything) since we no longer supplor python 3.6.